src/Data/AudienceFolder.elm

```diff
@@ -64,6 +83,18 @@ audienceFoldersJSON =
                 "name": "New Group 2",
                 "curated": false,
                 "parent": 3110
+            },
+            {
+                "id": 3111,
+                "name": "New Group 2 - Duplicated",
+                "curated": false,
+                "parent": 3110
+            },
+            {
+                "id": 12345,
+                "name": "Parent doesn't exist",
+                "curated": false,
+                "parent": 42
```

Comment:
I like your thinking about invalid states here, but I also think, that some logic and cases should be handled only in one place and in this case on API side so you will never get such an invalid data. This means less code on FE. But we do not live in ideal world so 👍